test/unit/math/src/Buffer.hpp

```diff
@@ -56,7 +56,7 @@ namespace mathtest
         // Constructor needs to initialize all Buffer.
         Buffer(DevAcc const& devAcc)
             : devHost{alpaka::getDevByIdx(platformHost, 0)}
-            , hostBuffer{alpaka::allocMappedBufIfSupported<TData, Idx>(devHost, platformAcc, Tcapacity)}
+            , hostBuffer{alpaka::allocMappedBufIfSupported<TData, Idx>(devHost, platformHost, Tcapacity)}
```

Comment:
why this change ?

Reply:
`allocMappedBufIfSupported` is supposed to allocate host memory, which is visible by the devices on the given platform if the mapped allocation is supported.

Calling `allocMappedBufIfSupported` with the host platform will allocate standard host memory.

Reply:
The code crashed during this allocation and I thought that was the reason, looking into it a bit more I've located the crash in the line `auto ctx = platform.syclContext();` (strange). I'll go on with the investigation of the crash and try to fix it.

Reply:
`platformAcc` was not initialized correctly, I've passed it to the constructor of `Buffer`. I've done the same for `platformHost` to be safe.